package.json

```diff
@@ -42,8 +42,8 @@
     "test.dist": "node scripts --validate-build",
     "test.end-to-end": "cd test/end-to-end && npm ci && npm test && npm run test.dist",
     "test.jest": "jest",
-    "test.karma": "cd test/karma && npm run karma",
-    "test.karma.prod": "cd test/karma && npm run karma.prod",
+    "test.karma": "cd test/karma && npm ci && npm run karma",
+    "test.karma.prod": "cd test/karma && npm ci && npm run karma.prod",
```

Comment:
Should we be running `test.karma.prod` in CI as well (i.e. change this to `npm run test.karma.prod`)?
https://github.com/ionic-team/stencil/blob/ccf4c57a619598b6f13cf485747390ce7ab065f2/.github/workflows/main.yml#L71

Reply:
👍 done in https://github.com/ionic-team/stencil/pull/2956/commits/4bc1011d76b038f51fad1c9ab6ae4effb2e06a72